NEWS.md

```diff
@@ -1,3 +1,14 @@
+# New in snapd 2.61.1:
+* Added prerequisite tracker for self-contained snap sets
```

Comment:
Not sure if we should say anything about this. My understanding is it is part of the larger default-provider story, which I am not 100% sure where we are.

Reply:
Hmm I don't know if this line should be here, it's not really outwards facing

Reply:
Agreed.

Reply:
Maybe somethink like "Do not require default provider on image building if alternative providers are included"

Reply:
what alfonso wrote, that's the user side story of those changes:

* Stop requiring default provider snaps on image building and first boot if alternative providers are included and available

Reply:
Added proposed NEWS entry and modified change logs accordingly.